src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -0,0 +1,489 @@
+# intracranial Electroencephalography (iEEG)
+
+Support for iEEG was developed as a [BIDS Extension Proposal](../06-extensions.md#bids-extension-proposals).
+Please cite the following paper when referring to this part of the standard in
+context of the academic literature:
+
+> Holdgraf, C., Appelhoff, S., Bickel, S., Bouchard, K., D'Ambrosio, S.,
+> David, O., … Hermes, D. (2018, December 13). BIDS-iEEG: an extension to the
+> brain imaging data structure (BIDS) specification for human intracranial
+> electrophysiology. [https://doi.org/10.31234/osf.io/r7vc2](https://doi.org/10.31234/osf.io/r7vc2)
+
+## iEEG recording data
+
+Template:
+
+```Text
+sub-<label>/
+  [ses-<label>]/
+    ieeg/
+      sub-<label>[_ses-<label>]_task-<task_label>[_acq-<label>][_run-<index>]_ieeg.<manufacturer_specific_extension>
+      sub-<label>[_ses-<label>]_task-<task_label>[_acq-<label>][_run-<index>]_ieeg.json
+```
+
+The iEEG community uses a variety of formats for storing raw data, and there is
+no single standard that all researchers agree on. The allowed file formats for
+iEEG data in BIDS are divided into two groups: _recommended_ formats that are
```

Comment:
iEEG uses "recommended" and "accepted" file formats while EEG uses "official" and "unofficial" - could this language be harmonized? For example using the established terms: "Its MANDATORY to use one of the (x,y,z) formats, but it's RECOMMENDED to use (x,y)"?

Reply:
good point, thanks! I use MUST and RECOMMENDED now.